opentelemetry-sdk/src/opentelemetry/sdk/metrics/__init__.py

```diff
@@ -366,11 +352,13 @@ def __init__(
         instrumentation_info: "InstrumentationInfo",
     ):
         self.instrumentation_info = instrumentation_info
-        self.batcher = UngroupedBatcher(source.stateful)
+        self.batcher = Batcher(True)
```

Comment:
is this still supposed to be `Batcher(source.stateful)` or is there a reason why its always stateful now?

Reply:
We removed the concept of "Stateful" on a processor in https://github.com/open-telemetry/opentelemetry-go/pull/840. Now the stateful property is per-instrument and depends on the requested kind of export.

Reply:
Will do sdk changes in a different PR.